INSTALL.md

```diff
@@ -123,6 +136,19 @@ settings that normally require modification.
 Start the server with `sudo systemctl start owntone` and check that it is
 running with `sudo systemctl status owntone`.
 
+To run multiple instances of owntone on a server, you should copy
```

Comment:
Added benefit of having it in the README is that this repetition is avoided

Reply:
Fair enough.  I did leave the `systemctl` commands here.